src/Stratis.Bitcoin/Base/Deployments/Models/ThresholdStateModel.cs

```diff
@@ -50,6 +51,18 @@ public class ThresholdStateModel
         [JsonProperty(PropertyName = "votes")]
         public int Votes { get; set; }
 
+        /// <summary>
+        /// The total number of blocks in the last confirmation window.
+        /// </summary>
+        [JsonProperty(PropertyName = "blocks")]
+        public int Blocks { get; set; }
+
+        /// <summary>
+        /// A summary of block version counts in the last confirmation window.
+        /// </summary>
+        [JsonProperty(PropertyName = "versions")]
+        public Dictionary<string, int> HexVersions { get; set; }
```

Comment:
This is not necessarily a major issue, but the block explorer shows block versions as integers.

Reply:
Its easier to identify the individual activations in a hex number.

Reply:
See my comment this should be on a class outside of `ThresholdStateModel`

Reply:
>We could have both if its an issue

I dont think its a big issue also not a bad idea to have both